source/server/BUILD

```diff
@@ -0,0 +1,27 @@
+load(
+    "@envoy//bazel:envoy_build_system.bzl",
+    "envoy_cc_library",
+)
+
+package(default_visibility = ["//visibility:public"])
+
+envoy_cc_library(
+    name = "http_test_server_filter_lib",
+    srcs = ["http_test_server_filter.cc"],
+    hdrs = ["http_test_server_filter.h"],
+    repository = "@envoy",
```

Comment:
Why do we need this `@envoy` business?

Reply:
Without it we get a batch of errors. the first one looks like
```
ERROR: /home/oschaaf/code/envoy-perf-vscode/nighthawk/test/server/BUILD:8:1: no such target '//bazel:windows_x86_64': target 'windows_x86_64' n
ot declared in package 'bazel' defined by /home/oschaaf/code/envoy-perf-vscode/nighthawk/bazel/BUILD and referenced by '//test/server:http_test_server_filter_integration_test'
```